wee_slack.py

```diff
@@ -2694,6 +2694,8 @@ def modify_buffer_line(buffer, new_line, timestamp, time_id):
         # hold the structure of a line and of line data
         struct_hdata_line = w.hdata_get('line')
         struct_hdata_line_data = w.hdata_get('line_data')
+        # keep track of the number of lines with the matching time and id
+        number_of_matching_lines = 0
 
         while line_pointer:
             # get a pointer to the data in line_pointer via layout of struct_hdata_line
```

Comment:
i don't know why there's an if there. Maybe we should have a lightweight assert-like functionality (but one which doesn't stop weechat completely) for invariants that must be true for us to continue?